src/getHumanAge.test.js

```diff
@@ -7,4 +7,49 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
```

Comment:
not bad, but pay attention to your descriptions

do not write "should return 0, 0..."
better to say "should return [0, 0]"

check it everywhere